nx-X11/programs/Xserver/hw/nxagent/Events.c

```diff
@@ -4107,7 +4107,7 @@ void nxagentSynchronizeExpose(void)
       {
         #ifdef TEST
         fprintf(stderr, "nxagentSynchronizeExpose: Going to call miWindowExposures"
-                    " for window [%ld] - rects [%ld].\n", nxagentWindow(pWin),
```

Comment:
Also okay, only used in this message.

I guess we can keep this commit as-is. Retracting my previous note to delete it.